pages/06.contribute/10.packaging_apps/60.advanced/20.config_panels/config_panels.md

```diff
@@ -162,10 +162,10 @@ ynh_app_config_run $1
 
 A question's getter is the function used to read the current value/state. Custom getters are defined using bash functions called `getter__QUESTION_SHORT_KEY()` which returns data through stdout.
 
-Stdout can generated using one of those formats:
- 1) either just the raw value,
- 2) or a yaml, containing the value and other metadata and properties (for example the `style` of an `alert`, the list of available `choices` of a `select`, etc.)
-
+Stdout can be generated using one of those formats:
+  1. either just the raw value,
+  2. or a yaml, containing the value and other metadata and properties (for example the `style` of an `alert`, the list of available `choices` of a `select`, etc.)
+Note that in the first case, if the raw value contains any yaml-senstive character (e.g. `#` which is interpreted as a comment in yaml), make sure it is returned between escaped quotes (cf. dedicated example below) as it will be [converted to yaml](https://github.com/YunoHost/issues/issues/2501) by Yunohost core in the end.
```

Comment:
```suggestion
Note that in the first case, if the raw value contains any yaml-senstive character (e.g. `#` which is interpreted as a comment in yaml), make sure it is returned between escaped quotes (cf. dedicated example below) as it will be [converted to yaml](https://github.com/YunoHost/issues/issues/2501) by YunoHost core in the end.
```